desktop/main/sources/fetchDiscovery.ts

```diff
@@ -23,12 +23,8 @@ import {
 import { handleIPC, handlerResult, makeSubscription } from '../rx.utils';
 import { downloadNodeConfig } from '../NodeConfig';
 
-export const fromNetworkConfig = (v) =>
-  from(downloadNodeConfig(v.conf)).pipe(
-    retry(3),
-    delay(200),
-    catchError(() => of([]))
-  );
+export const fromNetworkConfig = (net: Network) =>
+  from(downloadNodeConfig(net.conf)).pipe(retry(3), delay(200));
```

Comment:
Are you sure that getting rid of falling back to `[]` won't break the case, when there are no available networks (or internet connection) and the User wants to log in to check out his address or smth like this?

I remember I did this specifically for this case, actually.